internal/controller/drplacementcontrolvolsync.go

```diff
@@ -39,15 +40,12 @@ func (d *DRPCInstance) ensureVolSyncSetup(srcCluster string) error {
 	d.setProgression(rmn.ProgressionEnsuringVolSyncSetup)
 
 	// Create or update the destination VRG
-	err := d.createOrUpdateVolSyncDestManifestWork(srcCluster)
+	opResult, err := d.createOrUpdateVolSyncDestManifestWork(srcCluster)
 	if err != nil {
 		return err
 	}
 
-	vrgMWCount := d.mwu.GetVRGManifestWorkCount(rmnutil.DRPolicyClusterNames(d.drPolicy))
```

Comment:
Seems, that the function GetVRGManifestWorkCount is not in use anymore (it was the only place) and can be deleted,

Reply:
Ack.